src/status_im/contexts/wallet/connected_dapps/disconnect_dapp/view.cljs

```diff
@@ -0,0 +1,34 @@
+(ns status-im.contexts.wallet.connected-dapps.disconnect-dapp.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [status-im.contexts.wallet.connected-dapps.disconnect-dapp.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn view
+  [{:keys [wallet-account dapp on-disconnect]}]
+  (let [{:keys [avatar name]} dapp
+        {:keys [color]}       wallet-account]
```

Comment:
to improve performance, might be better to pass only the props it needs (avatar, name, color, on-disconnect) instead of the entire maps

Reply:
Updated it to accept `customization-color` instead of `wallet-account`, but passing `dapp` here makes sense because it's literally a map of just `avatar`, `name` and `url` & it makes sense to be passed here also for indicating that `disconnect-dapp` component requires `dapp` to be passed.